dev/deps/spark-deps-hadoop-3-hive-2.3

```diff
@@ -227,8 +228,8 @@ netty-transport-native-kqueue/4.1.115.Final/osx-x86_64/netty-transport-native-kq
 netty-transport-native-unix-common/4.1.115.Final//netty-transport-native-unix-common-4.1.115.Final.jar
 netty-transport/4.1.115.Final//netty-transport-4.1.115.Final.jar
 objenesis/3.3//objenesis-3.3.jar
-okhttp/3.12.12//okhttp-3.12.12.jar
-okio/1.17.6//okio-1.17.6.jar
+okhttp/3.14.2//okhttp-3.14.2.jar
```

Comment:
After removing the mandatory specification of `okio` dependency, the version of `esdk-obs-java-3.20.4.2` dependency was restored as follows:
https://repo1.maven.org/maven2/com/huaweicloud/esdk-obs-java/3.20.4.2/esdk-obs-java-3.20.4.2.pom